terraform/mgmtcluster.tf

```diff
@@ -45,21 +46,40 @@ final_message: "The system is finally up, after $UPTIME seconds"
 package_update: true
 package_upgrade: true
 write_files:
-  - encoding: b64
-    content: ewogICJtdHUiOiAxNDAwCn0K # set mtu 1400
+  - content: |
+      {
+        "mtu": ${var.kind_mtu}
+      }
     owner: root:root
     path: /tmp/daemon.json
     permissions: '0644'
+  - content: |
+      [Unit]
+      Description=Docker Set MTU to ${var.kind_mtu}
+      After=docker.service
+      Requires=docker.socket
+      
+      [Service]
+      Type=oneshot
+      ExecStart=/bin/ip link set dev docker0 mtu ${var.kind_mtu}
+      
+      [Install]
+      WantedBy=multi-user.target
```

Comment:
This relies on implementation details of docker (i.e. the interface name) and it also seems racey w.r.t. containers starting (and thus using the wrong initial MTU) and this unit file running.

Also, it only uses `After=docker.service` as synchronization and I don't think there's any guarantee that the network has been created when the service is up.

Reply:
Jonas, thanks for the review!
Good points.
I will do some extra testing -- my current belief is that this systemd unit can be dropped again, as the /etc/docker/daemon.json is still effective.

Reply:
As suspected: The systemd unit file is not required. PR #57 drops it. Review welcome!